presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoRpmControlUtils.java

```diff
@@ -138,6 +138,7 @@ public static Response restartUsingRpm()
 
     /**
      * The status api can be used to ascertain whether presto is installed and running.
+     * If Presto is installed, this method would return Presto version
```

Comment:
Javadocs should be a _statement_ of functionality, not a description.

The summary line should read something like "Get a response representing the status of the managed Presto node". (If more descriptive lines are present, each paragraph (and the summary) should be separated by a line containing only `<p>`.)

All occurences of "would" should be replaced by "will", or better yet, "would return" by "returns".